docs/admin/deployment/onboarding_admins.rst

```diff
@@ -31,60 +31,56 @@ To set up AW2, follow these steps:
    and `set an admin password on the welcome screen <https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/>`__
 2. Ensure that Tails and the SecureDrop version on AW1 are up-to-date.
    If not, update now by following the :ref:`most recent upgrade guide <latest_upgrade_guide>`.
-3. Insert the empty AW2 USB drive.
-4. Launch the Tails installer (**Applications ▸ Tails ▸ Tails Installer**) and install Tails on AW2.
+3. Log into the *Journalist Interface* using your admin credentials, and create
+    a new user account with admin rights. Record its passphrase securely;
+    you will add it to the password manager on AW2.
+
+    (You will need to on-board the new admin's 2FA device to complete this step.
+    If this is not possible yet, you can defer it until later.)
+4. Insert the empty AW2 USB drive.
+5. Launch the Tails installer (**Applications ▸ Tails ▸ Tails Installer**) and install Tails on AW2.
    This will delete all data on the AW2 USB drive.
-5. Shut down AW1.
-6. Boot AW2.
-7. Configure its persistent volume (**Applications ▸ Tails ▸ Configure persistent volume**).
+6. Shut down AW1.
+7. Boot AW2.
+8. Configure its persistent volume (**Applications ▸ Tails ▸ Configure persistent volume**).
```

Comment:
Consider the use of the clone persistent volume option in the tails installer instead - downside is that the entirety of AW1 gets copied, which may not be desirable. Upside is a probable further reduction in complexity.

Reply:
Sure, I'll aim for #491 while I'm here.

Reply:
Clarified this morning: I'm *not* going to aim for #491 while I'm here, since @nathandyer plans to tackle that.  (@nathandyer, probably you'll want to start from this branch to avoid duplicating or conflicting work, then rebase from `main` once this is merged.)